### DISCUSSION TITLE
更新2.8版本后文章发布失败

### DISCUSSION THREAD

**Discussion by dengchangdong** [Qexo/Qexo#375] [General]
博客程序：hugo
Qexo 版本： 2.8.0
出现问题：新增文章或者编辑文章，点击发布无任何提示，“保存草稿”又一切正常。
domian/edit.html 页面控制台报如下错误：
`Uncaught TypeError: Cannot read properties of null (reading 'classList')
    at index.min.js:1:237617
    at NodeList.forEach (<anonymous>)
    at e.addCaret (index.min.js:1:237575)
    at e.addToUndoStack (index.min.js:1:235226)
    at index.min.js:1:176311`

**Comment by am-abudu** [comment]
在2.8.1版本中该问题已修复。如果已经出现了这个问题，请在 设置->文章截取设置 中点击一次保存即可

**Reply by dengchangdong** [reply]
> 在2.8.1版本中该问题已修复。如果已经出现了这个问题，请在 设置->文章截取设置 中点击一次保存即可

谢谢大佬！
